operations/node_management/Configure_NTP_on_NCNs.md

```diff
@@ -4,59 +4,79 @@ The management nodes serve Network Time Protocol (NTP) at stratum 10, except for
 
 Until an upstream NTP server is configured, the time on the NCNs may not match the current time at the site, but they will stay in sync with each other.
 
-**Topics**
-   * [Fix BSS Metadata](#fix-bss-metadata)
-   * [Fix broken configs](#fix-broken-configs)
+## Topics
+
+* [Fix BSS Metadata](#fix-bss-metadata)
+* [Fix broken configurations](#fix-broken-configurations)
 
 <a name="fix-bss-metadata"></a>
 
-###### Fix BSS Metadata
+## Fix BSS Metadata
+
+If nodes are missing metadata for NTP, you will be required to generate the data using `csi` and your system's `system_config.yaml`.
 
-If nodes are missing metadata for NTP, you will be required to generate the data using `csi` and your system's `system_config.yaml`. If you do not have your seed data in the `system_config.yaml` then you will need to open a ticket to help generate the NTP data.
+If you do not have your seed data in the `system_config.yaml` then you will need to open a ticket to help generate the NTP data.
 
 The following steps are structured to be executed on one node at a time. However, step #3 will generate all relevant files for each node. If multiple nodes are missing NTP data in BSS, you can apply this fix to each node.
 
-1. Update system_config.yaml to have the correct NTP settings:
+1. Update `system_config.yaml` to have the correct NTP settings:
+
     ```yaml
     ntp-servers:
       - ncn-m001
       - time.nist.gov
     ntp-timezone: UTC
     ```
-2. Generate new configs:
+
+2. Generate new configurations:
+
     ```bash
     ncn# csi config init
     ```
+
 3. In the newly created `system/basecamp` directory, copy in and execute the metadata script that is included in the upgrade scripts of this documentation:
+
     ```bash
     ncn# ./upgrade_ntp_timezone_metadata.sh
```

Comment:
Do they need to copy the script there? Or is it enough to run the script from inside the system/basecamp directory? If that is good enough, then I think this suggestion works:
```suggestion
    ncn# cd system/basecamp && /usr/share/doc/csm/upgrade/scripts/upgrade_ntp_timezone_metadata.sh
```

Reply:
Fixed.